aws-test/tests/aws_securityhub_product/test-get-query.sql

```diff
@@ -0,0 +1,3 @@
+select name, product_arn, company_name, description, categories, activation_url
+from aws_securityhub_product
+where product_arn = 'arn:aws:securityhub:us-east-1:324264561773:product/guardicore/aws-infection-monkey';
```

Comment:
why we are hardcoding the full `product_arn` , it will fail if aws account is changed.

Reply:
this is the default products provided by aws